doc/cookbook/index.md

```diff
@@ -0,0 +1,148 @@
+# The SeqAn Cookbook {#cookbook}
+
+[TOC]
+
+This document provides example recipes on how to carry out particular tasks using the SeqAn functionalities in C++.
+Please note that these recipes are not ordered. You can use the links in the table of contents or the search function
+of your browser to navigate them. 
+It will take some time, but we hope to expand this document into containing numerous great examples. If you have suggestions for how to improve the Cookbook and/or examples you would like included, please feel free to contact us.
```

Comment:
```suggestion
of your browser to navigate them. 

It will take some time, but we hope to expand this document into containing numerous great examples. 
If you have suggestions for how to improve the Cookbook and/or examples you would like included, 
please feel free to contact us.
```
line length :nail_care: